CRFModuleValidation/Cardio_12MT.json

```diff
@@ -21,33 +40,42 @@
                            "identifier"   : "wearingFitbit",
                            "type"         : "instruction",
                            "title"        : "Wearing your fitbit?",
-                           "imageBefore"  : "fitbitCheckIcon",
                            "text"         : "Before continuing on with the 12 minute test, please make sure you have your fitbit on your wrist.",
-                           "actions"           : { "goForward": { "buttonTitle": "It’s on" }}
+                           "image"        : { "imageName": "fitbitCheckIcon", "placementType": "topBackground" },
+                           "colorTheme"   : { "backgroundColor" : "perrywinkle", "usesLightStyle" : true },
+                           "actions"      : { "goForward": { "buttonTitle": "It’s on" }},
+                           "viewTheme"    : { "viewIdentifier": "Instruction", "storyboardIdentifier": "ActiveTaskSteps" }
                            },
                            {
                            "identifier"   : "volumeUp",
                            "type"         : "instruction",
                            "title"        : "Volume turned up?",
-                           "imageBefore"  : "volumeCheckIcon",
                            "text"         : "Bring your phone with you and turn up your phone volume so you can hear the instructions while you are moving.",
-                           "actions"           : { "goForward": { "buttonTitle": "It’s turned on" }}
+                           "image"        : { "imageName": "volumeCheckIcon", "placementType": "topBackground" },
+                           "colorTheme"   : { "backgroundColor" : "perrywinkle", "usesLightStyle" : true },
+                           "actions"      : { "goForward": { "buttonTitle": "It’s turned on" }},
```

Comment:
should it say "It's turned up" ?

Reply:
Maybe? Personally, I like the fitbit button best. "It's on" ;)